ivy/functional/frontends/torch/linalg.py

```diff
@@ -189,6 +189,38 @@ def solve(input, other, *, out=None):
     return ivy.solve(input, other, out=out)
 
 
+@to_ivy_arrays_and_back
```

Comment:
why there is this change to the torch frontend end?

Reply:
that was another task I was working on and commited through the same branch should make it saperated?
then I will make another PR
would it be ok